### DISCUSSION TITLE
گزینه تارگت اسپید چه تاثیری روی نتیجه اسکن دارد ؟

### DISCUSSION THREAD

**Discussion by Persianfriend** [MortezaBashsiz/CFScanner#309] [Q&A] [answered]
سلام و خسته نباشید بابت این برنامه فوق‌العاده،

ممنون میشم اگر در مورد تاثیر Targt Speed روی نتیجه اسکن توضیح مختصری ارائه بفرمایید. 

سپاس

**Comment by goingfine** [comment, marked as answer]
این یعنی حداقل سرعتی که روی IP میتونید بگیرید باید جه سرعتی باشه.
هرچی اینترنتتون پرسرعت تر باشه میتونید روی عدد بالاتری بزارید تا نتیجه دقیق تری بگیرید و همچنین برعکس هرچه سرعت اینترننتون کمتر باشه روی عدد پایین تری میزارید تا هم سرعت تست بالاتر بره و هم IP های سالم رو از دست ندید

**Reply by Persianfriend** [reply]
بسیار بسیار ممنونم، با آرزوی بهترینها برای شما
